client/state/initial-state.js

```diff
@@ -48,7 +48,24 @@ function deserialize( state ) {
 	return reducer( state, { type: DESERIALIZE } );
 }
 
-function loadInitialState( initialState ) {
+function addSympathy( initialStateLoader ) {
+	if ( 'development' !== process.env.NODE_ENV || ( Math.random() > 0.5 ) ) {
+		return initialStateLoader;
+	}
+
+	console.log( 'Skipping initial state load to recreate first-load experience.' ); // eslint-disable-line no-console
+
+	try {
+		localStorage.clear();
+		indexedDB.deleteDatabase( 'calypso' );
```

Comment:
@gwwar I wouldn't think it matters that it's async because we're not loading it anyway and I would imagine if there are multiple `indexedDB` calls that they would form a queue and the second wouldn't be processed before the first, maybe? sound right?